Example/Tests/Helpers/SpyTracker.swift

```diff
@@ -9,32 +9,44 @@
 import Foundation
 
 class SpyTracker {
-    var spies: [String: [String: String]?] = [:]
+    var spies: [String: [[String: String]?]] = [:]
 
     func track(functionName: String, args: [String: String]? = nil) {
-        spies[functionName] = args
+        if spies[functionName] == nil {
+            spies[functionName] = []
+        }
+
+        spies[functionName]?.append(args)
```

Comment:
nit: you could do this instead to make it simpler.

```suggestion
        spies[functionName, default: []].append(args)
```